examples/jekyll/Gemfile.lock

```diff
@@ -52,21 +52,22 @@ GEM
       jekyll-seo-tag (~> 2.1)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
-    public_suffix (4.0.6)
-    rb-fsevent (0.10.4)
+    public_suffix (5.0.4)
+    rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    rexml (3.2.4)
-    rouge (3.26.0)
+    rexml (3.2.6)
+    rouge (3.30.0)
     safe_yaml (1.0.5)
     sassc (2.4.0)
       ffi (~> 1.9)
     terminal-table (2.0.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
-    unicode-display_width (1.7.0)
+    unicode-display_width (1.8.0)
+    webrick (1.8.1)
 
 PLATFORMS
-  x86_64-linux
+  ruby
```

Comment:
What does the change in platform do here? That is, what's the difference between `x86_64-linux` and `ruby`?

Reply:
I'd also like to better understand. The docs don't say that `x86_64-linux` is a valid value, but maybe it used to be.
https://bundler.io/v2.3/man/gemfile.5.html#PLATFORMS

Reply:
`[PLATFORMS](https://guides.rubygems.org/specification-reference/#platformplatform)` is used for gems with native binary bindings that get pre-compiled, so gems specify which versions need to be installed (sometimes multiples and only one gets actually `required` depending on what platform the interpreter is running on at the moment).

These get written to the lockfile when running `bundle`. [Here's a nice little explainer](https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/#platforms)

So, it's common to have values like

```
PLATFORMS
  x86_64-linux
  arm64-darwin
  ruby
```

If you have a mix of pure ruby gems and gems with some pre-compiled binaries and you deploy to a linux system but your developers work on their Macbooks.

In this case, I'm guessing the final gem with a pre-compiled dependency was either removed or a newer version has a pure-Ruby implementation.